src/subclass/boxed.rs

```diff
@@ -45,10 +45,9 @@ pub fn register_boxed_type<T: BoxedType>() -> ::Type {
         use std::ffi::CString;
 
         let type_name = CString::new(T::NAME).unwrap();
-        assert_eq!(
-            gobject_sys::g_type_from_name(type_name.as_ptr()),
-            gobject_sys::G_TYPE_INVALID
-        );
+        if gobject_sys::g_type_from_name(type_name.as_ptr()) != gobject_sys::G_TYPE_INVALID {
+            panic!("Type {} has already be registered", type_name.to_str().unwrap());
```

Comment:
s/be/been/ :) Also in the other place below

Reply:
That's what you get with the first patch in the morning ;)